packages/design-system/modal/modal.container.web.tsx

```diff
@@ -61,27 +57,47 @@ const ModalContainerComponent = forwardRef<ModalMethods, ModalContainerProps>(
       []
     );
 
-    useEscapeKeydown((event) => {
-      event.preventDefault();
-
-      return disableBackdropPress ? noop() : onClose?.();
-    });
-
-    // Prevent scrolling the body when the modal is open
-    useLockBodyScroll();
-
     return (
-      <FocusTrap aria-modal>
```

Comment:
@intergalacticspacehighway I have removed this component from the Modal because I didn't find it useful now, and it's cuasing a bug in ourpject, you can see [this thread](https://showtime-rq88331.slack.com/archives/C02Q0E4PBD0/p1682945414129099). 
Maybe it's because we have already upgraded RNW to v18 and switched to NativeWind. I think the previous issue no longer exists, or could you check it agian in this PR?

Reply:
@alantoa i think [this line](https://github.com/showtime-xyz/showtime-frontend/blob/staging/packages/design-system/modal/modal.backdrop.web.tsx#L19) is causing for the bug. Can you check by removing it? I tried and it fixed the issue. I am not sure why onTouchEnd was used though